src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionStringBuilder.cs

```diff
@@ -38,26 +42,34 @@ private enum Keywords
             MinPoolSize,
             MaxPoolSize,
             PoolBlockingPeriod,
+#if NETFRAMEWORK
             ConnectionReset,
+#endif
             MultipleActiveResultSets,
             Replication,
             ConnectTimeout,
             Encrypt,
             TrustServerCertificate,
             LoadBalanceTimeout,
+#if NETFRAMEWORK
             NetworkLibrary,
+#endif
             PacketSize,
             TypeSystemVersion,
             Authentication,
             ApplicationName,
             CurrentLanguage,
             WorkstationID,
             UserInstance,
+#if NETFRAMEWORK
             ContextConnection,
+#endif
             TransactionBinding,
             ApplicationIntent,
             MultiSubnetFailover,
+#if NETFRAMEWORK
             TransparentNetworkIPResolution,
+#endif
```

Comment:
Is the order of keywords important or could they be reordered so that all the netfx ones are in a single block?

Reply:
As you know, the Enum order would be important if we used the member values. I didn't check its references and tried to keep it's order like it was. I'll check to find whether we could group them together.

Reply:
done